packages/playwright-core/src/client/connection.ts

```diff
@@ -98,8 +98,14 @@ export class Connection extends EventEmitter {
     return await this._rootObject.initialize();
   }
 
-  pendingProtocolCalls(): ParsedStackTrace[] {
-    return Array.from(this._callbacks.values()).map(callback => callback.stackTrace).filter(Boolean) as ParsedStackTrace[];
+  pendingProtocolCalls(): String {
+    const lines: string[] = [];
+    for (const call of this._callbacks.values()) {
+      if (!call.apiName)
+        continue;
+      lines.push(`  - ${call.apiName}\n${stringifyStackFrames(call.frames)}\n`);
```

Comment:
Shouldn't this be `call.frames.slice(1)`? I don't think we had the full stack here before.